src/zcash/JoinSplit.cpp

```diff
@@ -137,16 +137,18 @@ class JoinSplitCircuit : public JoinSplit<NumInputs, NumOutputs> {
         } catch (...) {
             return false;
         }
-    }
+    
+}
```

Comment:
Undo this change and then squash, please.

Reply:
Should I keep your merge commit, or remove the `const`s in my commit and force push?

Reply:
Squash how far? Should I also squash your merge commit?

Reply:
Yes.